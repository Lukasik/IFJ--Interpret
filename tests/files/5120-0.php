<?php

// strval

function f()
{}

$x=f(2,3);
$null=strval($x);


function soucin($x,$y)
{
	return $x*$y;
}

function cmp($x,$y)
{
	return $x===$y;
}

$x=cmp("aaa",1);
$boolnic=strval($x);

$x=soucin(7,2);
$int14=strval($x);


$q=0-11;
$q2=0-0.5;
$x=soucin($q,$q2);
$double=strval ($x);


$x="a"."b";
$str=strval($x);

$x=put_string($null,$boolnic,$int14,"\n",$double,"\n",$str);


// null ""   boolnic ""   int14  "14"  double 5.5  str "ab"



