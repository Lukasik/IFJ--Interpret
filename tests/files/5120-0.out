14
5.5
ab